_pages/index.html

```diff
@@ -16,18 +16,28 @@
 </div>
 
 <div class="emphasize">
-  {% if site.phase==1 %}
-    <h2>Tickets for 2017 will go on sale in Hilary term, like us on <a href="https://facebook.com/insciteco/">Facebook</a> for updates.</h2>
+
+  {% if site.phase == 1 %}
+  
+  <h2>
+    Tickets for 2017 will go on sale in Hilary term, like us on <a href="https://facebook.com/insciteco/">Facebook</a> for updates.
+  </h2>
+  
   {% elsif site.phase==0 %}
+  
     <h3><b>80% off tickets!</b></h3>
     <h4 style="display:inline;"> Ticket prices have just been reduced from &pound;25 to </h4><h3 style="display:inline;">&pound;5!</h3>
     <br><br>
-    <a class="btn btn-success" href="http://inscite.co/attend/buy">Buy now</a>
+    <a class="btn btn-success" href="{{ site.baseurl }}/attend/buy">Buy now</a>
+    
   {% elsif site.phase == 2 %}
+  
     <h2>We are now accepting applications for speakers!</h2><br>
-    <a href="http://www.inscite.co/talk" class="button">Submit an application</a> or email us at <a href="mailto:speakers@inscite.co">speakers@inscite.co</a> if you have any questions.<br>
-    <h2>Deadline is Friday 4th of December 2015</h2>
+    <a href="{{ site.baseurl }}/talk" class="button">Submit an application</a> or email us at <a href="mailto:speakers@inscite.co">speakers@inscite.co</a> if you have any questions.<br>
```

Comment:
ooh, button class is better with: "btn btn-success" instead of button, but that's fine